packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -38,69 +38,85 @@ export const Settings = () => {
   // const secondaryEmails = account.emails.filter((email) => !email.isPrimary);
 
   return (
-    <main data-testid="settings-container" className="bg-grey-20 min-h-screen">
-      <h2>Settings</h2>
-      <section>
-        <h3>Profile</h3>
-
-        <UnitRowWithImage
-          header="Picture"
-          imageUrl={account.avatarUrl}
-          alt="Your avatar"
-          route="#"
-        />
-
-        {account.displayName && (
+    <>
+      <section className="mt-11">
+        <h2 className="font-bold ml-4 mb-4">Profile</h2>
+
+        <div className="bg-white tablet:rounded-xl shadow">
+          <UnitRowWithImage
+            header="Picture"
+            imageUrl={account.avatarUrl}
+            alt="Your avatar"
+            route="#"
+          />
+
+          <hr className="unit-row-hr" />
+
           <UnitRow
             header="Display name"
             headerValue={account.displayName}
             route="#"
           />
-        )}
-
-        <UnitRow header="Password" headerValue="********" route="#">
-          <p>Created {account.passwordCreated}</p>
-        </UnitRow>
-
-        {/*
-        /* TO DO: primary/secondary email section with
-        /* verified status and number of secondary emails
-        /* taken into account
-        */}
-        <UnitRow header="Primary email" headerValue={primaryEmail.email} />
-
-        <UnitRow
-          header="Secondary email"
-          headerValue={null}
-          {...{
-            revealModal,
-            modalRevealed,
-          }}
-        >
-          <p>Access your account if you can't log in to your primary email.</p>
-          <p>
-            Note: a secondary email won't restore your information—you'll need a{' '}
-            <a href="#recovery-key">recovery key</a> for that.
-          </p>
-
-          {modalRevealed && (
-            <Modal
-              onDismiss={hideModal}
-              onConfirm={onSecondaryEmailConfirm}
-              headerId={modalHeaderId}
-              descId={modalDescId}
-            >
-              <h2 id={modalHeaderId}>Verify primary email first</h2>
-              <p id={modalDescId}>
-                Before you can add a secondary email, you must verify your
-                primary email. To do this, you'll need access to{' '}
-                {primaryEmail.email}
-              </p>
-            </Modal>
-          )}
-        </UnitRow>
+
+          <hr className="unit-row-hr" />
+
+          <UnitRow
+            header="Password"
+            headerValueClassName="tracking-wider"
+            headerValue="••••••••••••••••••"
+            route="#"
+          >
+            <p className="text-grey-400 text-xs mobileLandscape:mt-3">
```

Comment:
I feel like I broke a convention making the breakpoints camelCase. Should we change them to dashes?

Reply:
Hmm well the only one we have is `mobileLandscape`, I think I actually prefer it that way because `mobile-landscape` might look less obvious as its own entity than `mobileLandscape` but if you strongly prefer `mobile-landscape` then I think that's fine (maybe we'd change it after this PR is merged) 👍

Reply:
desktopXl as well

Reply:
Ah, well, I'm fine with either, it doesn't bother me. :)